### ISSUE TITLE
Admin ability to add ChA profile to a mentor account

### ISSUE THREAD

**Issue by shaun-technovation** [Iridescent-CM/technovation-app#4409]
This ticket covers the basic core functionality of allowing PT to go to a mentors admin page and click a button to add a Chapter Ambassador profile to that mentor account. There will be additional tickets including:

- We need to send an email to the new ChA with information about their new account
- We need the ability to maintain a mentor mode for the new ChA
- We need to have an option to convert a ChA account back to just a mentor account

Notes: The most common use case is a mentor who becomes a Chapter Ambassador. For context, we will likely be adding a club ambassador role in the future and admins will also need to be able to convert that role to a ChA.

**Comment by dboyer** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Technovation-605271851956b1000ec2f24d/issues/Iridescent-CM/technovation-app/4409?planning-poker) with Zenhub @shaun-technovation @viviancan

**Comment by viviancan** [comment]
Confirming this is on QA and working as expected. I was able to add a ChA profile to an account with a mentor profile. I the mentor profile did not have a filled in/completed bio I received an error with the bio must be filled out error.

**Comment by dboyer** [comment]
Confirmed on QA:

- I see the button a mentor account to add a ChA profile
- Logged in as the new ChA and went to their mentor profile where I had a button to switch to ChA mode
<img width="356" alt="Screen Shot 2024-01-30 at 10 53 33 AM" src="https://github.com/Iridescent-CM/technovation-app/assets/516764/eb6f519a-3aac-43d4-bb75-b020a4f8872d">
- Was marked as pending and then when I approved the ChA account as an admin, I was able to see the full ChA account when I logged back in

Tested also on the preview site:

- Button to convert to a ChA was visible on the mentor's admin participant page
- Logged in as the new ChA and went straight to new ChA dashboard, with no interim pending step
- Was able to use the mentor mode button to switch back to the mentor dashboard

I see the button on a judge account if they also have a mentor profile but not if they have just a judge account. I think that's the right behavior. We'll need an option to remove the ChA profile for a user and return them to being just a mentor but that can be a different ticket.
